client/package.json

```diff
@@ -42,9 +45,12 @@
     "extract-text-webpack-plugin": "0.9.1",
     "file-loader": "^0.8.4",
     "html-webpack-plugin": "^1.6.1",
+    "imports-loader": "^0.6.5",
     "ip": "1.1.0",
+    "jquery": "^2.2.0",
```

Comment:
Why do we need to import JQuery explicitely? Isn't it a dependency of some other package? I don't think we use jquery anywhere